tools/debug_tools/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "debug_tools"
+version = "0.0.2"
+authors = ["Lyla <wallamide@gmail.com>"]
+edition = "2021"
+description = "Debugging tools for Emergence"
+publish = false
+license = "MIT OR Apache-2.0"
+
+[lib]
+name = "debug_tools"
+
+[dependencies]
+bevy = "0.9"
+bevy_console = { git = "https://github.com/RichoDemus/bevy-console", branch = "main" }
```

Comment:
Pick a commit for reproducible builds.

Reply:
this will be updated to the newest stable release on crates.io